content/index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Software For Humans
+title: We Design & Develop Web Apps For Humans
```

Comment:
Something like this makes sense to me, but remember the main place this shows up is in a potentially very small tab, so we want the most important/meaningful words at the front:

![We des…](https://dl.dropbox.com/s/9ahwey6q82c4x75/Screen%20Shot%202021-06-22%20at%2011.37.03%20AM.jpg)

Reply:
In such a small context as you show in your screenshot, maybe the only change I would make would to reorder "Develop & Design." The other option I thought about would be "Web App Design & Development for Humans," but then only "Web App" would be likely to show up. I think it may be a toss up as to whether "Develop" or "Web App" is most important/meaningful. We could remove the word "We," but I think in a bigger context the phrase "Design & Develop..." sounds like we're telling someone else to do a thing instead of explain what our services are -- like maybe we help other people design and develop this own web apps for humans. Those are my musings. Thoughts? Do you like the new image?
![oddsite](https://user-images.githubusercontent.com/8933115/122976663-d9833b00-d351-11eb-8e97-a67b8e17b037.jpg)

Reply:
yeah, the image is great. Could even get that tagline back on it maybe - just so it's able to stand alone? Out of curiosity, I checked out the CSS Tricks default, and it's similarly simple title & tagline: 

![](https://dl.dropbox.com/s/3vx631t8m9d9os8/Screen%20Shot%202021-06-22%20at%2012.14.13%20PM.jpg)

I agree with your thoughts. "Web App Design & Development" doesn't seem like it would need the "we" - but I don't know if "web app" is a batter start than "we develop" or not? "Custom web applications"? …???

Reply:
@mirisuzanne I went with "Custom Web App Design & Development For Humans" as the title. Here's the updated picture.
![oddsite](https://user-images.githubusercontent.com/8933115/122989906-967c9400-d360-11eb-90be-63dd97c347c2.jpg)